plugins/analysis/dashboard/parameters.go

```diff
@@ -15,14 +15,14 @@ const (
 	CfgBasicAuthUsername = "analysis.dashboard.basic_auth.username"
 	// CfgBasicAuthPassword defines the config flag of the analysis dashboard basic auth password.
 	CfgBasicAuthPassword = "analysis.dashboard.basic_auth.password"
-	// CfgFPCBufferSize defines the config flag of the analysis dashboard FPC (past conflicts) buffer size.
-	CfgFPCBufferSize = "analysis.dashboard.fpc.buffer_size"
+	// CfgMongoDBEnabled defines the config flag of the analysis dashboard to enable mongoDB.
+	CfgMongoDBEnabled = "analysis.dashboard.mongodb.enabled"
 	// CfgMongoDBUsername defines the config flag of the analysis dashboard mongoDB username.
 	CfgMongoDBUsername = "analysis.dashboard.mongodb.username"
-	// CfgMongoDBPassword defines the config flag of the analysis dashboard mongoDBpassword.
+	// CfgMongoDBPassword defines the config flag of the analysis dashboard mongoDB password.
 	CfgMongoDBPassword = "analysis.dashboard.mongodb.password"
-	// CfgMongoDBBindAddress defines the config flag of the analysis dashboard mongoDB binding address.
-	CfgMongoDBBindAddress = "analysis.dashboard.mongodb.bindAddress"
+	// CfgMongoDBHostAddress defines the config flag of the analysis dashboard mongoDB binding address.
+	CfgMongoDBHostAddress = "analysis.dashboard.mongodb.hostAddress"
```

Comment:
Does not have anything to do with the "analysis dashboard", maybe just rename to "analysis server"

Reply:
But mongoDB it's used by the dashboard plugin, not the analysis-server one.

Reply:
For me gathering the data about finalized vote contexts doesn't really have anything to do with the dashboard per se. My assumption is that storing that data is part of the analysis server whether or not the dashboard is enabled. What is your take on that?

Aka, the dashboard should only visualize whatever the analysis-server is gathering. However, I see now that autopeering data is also retained in the dashboard plugin which technically from my PoV is not correct.

Lets leave it as it is but maybe revisit this at a later point in time.

Reply:
Yeah I perfectly agree with you, we should, eventually, revisit that design and do as you are suggesting.